apps/synapse-oauth-signin/src/AppInitializer.tsx

```diff
@@ -12,18 +12,17 @@ const cookies = new UniversalCookies()
 function AppInitializer(
   props: React.PropsWithChildren<Record<string, unknown>>,
 ) {
-  const accountSitePrompted = cookies.get(
-    SynapseConstants.ACCOUNT_SITE_PROMPTED_FOR_LOGIN_COOKIE_KEY,
-  ) == 'true' // short-lived cookie
+  const accountSitePrompted =
+    cookies.get(SynapseConstants.ACCOUNT_SITE_PROMPTED_FOR_LOGIN_COOKIE_KEY) ==
+    'true' // short-lived cookie
   const urlSearchParams = new URLSearchParams(window.location.search)
-  const prompt =
-    accountSitePrompted ? 'none' : urlSearchParams.get('prompt')
+  const prompt = accountSitePrompted ? 'none' : urlSearchParams.get('prompt')
 
   let maxAge = undefined
   // check max age when re-establishing the session, not to auto-consent.
   const maxAgeURLParam = urlSearchParams.get('max_age')
   // SWC-5597: if max_age is defined, then return if the user last authenticated more than max_age seconds ago
-  if (maxAgeURLParam && parseInt(maxAgeURLParam)) {
+  if (!accountSitePrompted && maxAgeURLParam && parseInt(maxAgeURLParam)) {
```

Comment:
This is the only real change (the other changes are formatting, which are auto-formatted during commit so I'm not sure why they differ)